src/main.js

```diff
@@ -735,6 +735,8 @@ function showKills(simResult) {
     let dropsResultDiv = document.getElementById("simulationResultDrops");
     let newChildren = [];
     let newDropChildren = [];
+    let dropRateMultiplier = simResult.dropRateMultiplier;
```

Comment:
You only set this drop rate multiplier if there is a buff currently active, if equipment with a drop rate multiplier exists in the future, it wouldn't show up here unless the buff was active. You can get around this by setting the simResult.dropRateMultiplier regardless of if there is a buff or not, or a simple hack to get around this would be:
`if (dropRateMultiplier == 1) {dropRateMultiplier = 1 + player.combatDetails.combatStats.combatDropRate}`

Reply:
Set it to trigger the drop multiplier on simulation cleanup, rather than on buff activation.